py-polars/src/series/mod.rs

```diff
@@ -679,6 +679,14 @@ impl PySeries {
         self.series.clear().into()
     }
 
+    fn head(&self, n: Option<usize>) -> Self {
+        self.series.head(n).into()
+    }
```

Comment:
The input here is not an Option type (same for tail):

```suggestion
    fn head(&self, n: usize) -> Self {
        self.series.head(Some(n)).into()
    }
```